test/parallel/test-dgram-unref-in-cluster.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const dgram = require('dgram');
+const cluster = require('cluster');
+const assert = require('assert');
+
+if (common.isWindows)
+  common.skip('dgram clustering is currently not supported on Windows.');
+
+if (cluster.isPrimary) {
+  cluster.fork();
+} else {
+  const socket = dgram.createSocket('udp4');
+  socket.unref();
+  socket.bind();
+  socket.on('listening', common.mustCall(() => {
+    const sockets = process._getActiveHandles().filter((item) => item instanceof dgram.Socket);
```

Comment:
nit: since `process._getActiveHandles()` is doc-deprecated
```suggestion
    const sockets = process.getActiveResourcesInfo().filter((item) => item === 'UDPWrap');
```

Reply:
Fixed, thanks.